collections/misc/occurrencesearch.php

```diff
@@ -6,10 +6,10 @@
 $targetId = filter_var($_REQUEST['targetid'], FILTER_SANITIZE_NUMBER_INT);
 $collid = array_key_exists('collid', $_REQUEST) ? filter_var($_REQUEST['collid'], FILTER_SANITIZE_NUMBER_INT) : 0;
 $action = array_key_exists('action', $_POST) ? $_POST['action'] : '';
-$catalogNumber = array_key_exists('catalognumber',$_POST) ? filter_var($_POST['catalognumber'], FILTER_SANITIZE_STRING) : '';
-$otherCatalogNumbers = array_key_exists('othercatalognumbers',$_POST) ? filter_var($_POST['othercatalognumbers'], FILTER_SANITIZE_STRING) : '';
-$recordedBy = array_key_exists('recordedby',$_POST) ? filter_var($_POST['recordedby'], FILTER_SANITIZE_STRING) : '';
-$recordNumber = array_key_exists('recordnumber',$_POST) ? filter_var($_POST['recordnumber'], FILTER_SANITIZE_STRING) : '';
+$catalogNumber = array_key_exists('catalognumber',$_POST) ? $_POST['catalognumber'] : '';
```

Comment:
Do we want a variable labeled number defaulting to an empty string?

I tried to follow the logic outwards (e.g. to getNewDetItem in classes/OccurrenceEditorDeterminations.php), and it seems to be handled gracefully.

I guess this ok, right?